bin/setup_dependencies.sh

```diff
@@ -37,8 +37,8 @@ chmod -R 600 /home/ubuntu/.ssh
 chmod +x /home/ubuntu/.ssh
 
 # Disable root access
-passwd -l root
-echo 'SSH access for root disabled. You will need to connect as ubuntu.'
+# passwd -l root
```

Comment:
Sorry just got around to looking at this. The commits seem still to be in there. Also there seem to be other things in here. Can you clean this up and make it just the blue/green thing?